diff --git a/rllib/examples/learners/classes/intrinsic_curiosity_learners.py b/rllib/examples/learners/classes/intrinsic_curiosity_learners.py
index dd37dab0cb..acbbfd7e0c 100644
--- a/rllib/examples/learners/classes/intrinsic_curiosity_learners.py
+++ b/rllib/examples/learners/classes/intrinsic_curiosity_learners.py
@@ -150,10 +150,12 @@ class IntrinsicCuriosityModelConnector(ConnectorV2):
         with torch.no_grad():
             # Perform ICM forward pass.
             fwd_out = rl_module[ICM_MODULE_ID].forward_train(batch[DEFAULT_MODULE_ID])
+            rewards = fwd_out[Columns.INTRINSIC_REWARDS]
+            mitigatedReward = torch.where(rewards < 0.01, 0, rewards)
 
         # Add the intrinsic rewards to the main module's extrinsic rewards.
         batch[DEFAULT_MODULE_ID][Columns.REWARDS] += (
-            self.intrinsic_reward_coeff * fwd_out[Columns.INTRINSIC_REWARDS]
+            self.intrinsic_reward_coeff * mitigatedReward
         )
 
         # Duplicate the batch such that the ICM also has data to learn on.
